### ISSUE TITLE
High severity issues sent from GH regarding package-lock.json

### ISSUE THREAD

**Issue by zanodor** [oleeskild/digitalgarden#272]
I've seen these concerning `ws` and `pug`:
- ws affected by a DoS when handling a request with many HTTP headers
- Pug allows JavaScript code execution if an application accepts untrusted input

As much I've managed to garner, these issues would go away if the user adds the new versions addressing these vulnerabilites to the package-lock file.  

I wonder if I'm the only one having these so I thought I'd throw this up and also wonder how much troubleshooting is necessary to keep the template going...?

Cheers

**Comment by Curiosity432** [comment]
[html-minifier has high security vulneravility](https://github.com/mjmlio/mjml/issues/2856), and cannot install dependencies. I try to fix it with html-minifier-terser but have problems and cannot build locally.

Sames happens for me on GitHub running the workflow on GitHub actions gets an error.

```sh
 ╰─λ npm install
up to date, audited 527 packages in 1s
140 packages are looking for funding
  run `npm fund` for details

1 high severity vulnerability

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
```

@oleeskild you should update the dependencies of the template.

**Comment by Curiosity432** [comment]
Now it is fixed with the new udpate v1.61.3 https://github.com/oleeskild/digitalgarden/commit/693a35f864d53a225cda3b4210a8c76a9fbf95f7

Thank you very much @oleeskild. Now is working perfectly and there are 0 vulnerabilities detected by npm.

This issue can be closed.
